docs/pages/use-teleport/tsh.mdx

```diff
@@ -764,5 +764,55 @@ $ tsh status
   Extensions:         permit-X11-forwarding
 ```
 
+## Custom aliases and defaults
+
+You can define your own custom commands and set command-specific flag defaults by using the `aliases` section of `tsh` config.
+
+There are two configuration files:
+- global: by default `/etc/tsh.yaml`, unless overridden by `$TELEPORT_GLOBAL_TSH_CONFIG` env var.
+- user-specific: `$TELEPORT_HOME/config/config.yaml`, which by defaults resolves to `~/.tsh/config/config.yaml`.
+
+The user-specific config is merged with global config. In case of conflicts (i.e. same alias defined in both files), the user-specific config has higher priority.
```

Comment:
I think we should move this paragraph lower in the section. Readers don't know what an alias configuration looks like at this point in the section, and I think this would be good to know before we talk about merging configs.